spyder_unittest/widgets/tests/test_unittestgui.py

```diff
@@ -33,7 +31,8 @@ def test_run_tests_and_display_results(qtbot, tmpdir):
     qtbot.wait(1000) # wait for tests to run
```

Comment:
@jitseniesen in general I dont think this is a good idea. The better way to do this is with a signal and using this construct

http://pytest-qt.readthedocs.io/en/latest/signals.html

```python
    with qtbot.waitSignal(app.worker.finished, timeout=10000) as blocker:
        app.worker.start()
        # Test will block at this point until either the "finished" or the
        # "failed" signal is emitted. If 10 seconds passed without a signal,
        # SignalTimeoutError will be raised.

```